sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManager.kt

```diff
@@ -155,6 +157,26 @@ interface ViewAnnotationManager {
    */
   fun getViewAnnotationUpdateMode(): ViewAnnotationUpdateMode
 
+  /**
+   * Get the [CameraOptions] object bound to specified annotations on the MapView.
+   *
+   * Using this method with GlobeProjection might lead to incorrect results.
```

Comment:
how incorrect are they? I thought we will be introducing some sort of exception or we could return NULL if projection is globe

Reply:
I have tested it and `cameraForCoordinates` doesn't provide any exceptions, so no need to add exceptions here.

> how incorrect are they? 

For eg. if we select points that are far apart, when it calculates cameraOption center, it does fall in the place where both annotations are not visible.

Reply:
imo we should either support it fully - then remove the warning docs or do not support at all. Looking at `MapboxMap#cameraForCoordinateBounds` docs: 
```
This API isn't supported by Globe projection and will return a no-op result matching the world bounds.
```
I lean to having similar behaviour here